forage-android/src/main/java/com/joinforage/forage/android/pos/ForageTerminalSDK.kt

```diff
@@ -0,0 +1,190 @@
+     * using Track 2 data from a magnetic card swipe.
+     *
+     * @param track2Data The Track 2 data obtained from the magnetic stripe of the card.
+     * @param reusable Optional. Indicates whether the tokenized card can be
+     * reused for multiple transactions. Defaults to true.
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the operation.
+     * On success, returns a [PaymentMethod](https://docs.joinforage.app/reference/create-payment-method)
+     * token which can be securely stored and used for subsequent transactions. On failure,
+     * returns a detailed error response for proper handling.
+     */
+    suspend fun tokenizeCard(
+        track2Data: String,
+        reusable: Boolean = true
+    ): ForageApiResponse<String> {
+        logger.i(
+            "[POS] Tokenizing Payment Method via magnetic card swipe",
+            attributes = mapOf(
+                "reusable" to reusable
+            )
+        )
+
+        return ForageApiResponse.Success("TODO")
+    }
+
+    /**
+     * TODO: add comment here
+     */
+    suspend fun refundPosPayment(
+        params: RefundPosPaymentParams
+    ): ForageApiResponse<String> {
+        return ForageApiResponse.Success("TODO")
+    }
+
+    /**
+     * Checks the balance of a given PaymentMethod using a ForagePINEditText
+     *
+     * @param params The parameters required for balance inquiries, including
+     * a reference to a ForagePINEditText and PaymentMethod ref
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the operation.
+     * On success, returns an object with `snap` (SNAP) and `cash` (EBT Cash) fields, whose values
+     * indicate the current balance of each respective tender.
+     *
+     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    override suspend fun checkBalance(
+        params: CheckBalanceParams
+    ): ForageApiResponse<String> {
+        return ForageApiResponse.Success("TODO")
+    }
+
+    // ======= Same as online-only Forage SDK below =======
+
+    /**
+     * Captures a Forage Payment associated with an EBT card using a customer's EBT card PIN.
+     *
+     * @param params The parameters required for payment capture, including
+     * reference to a ForagePINEditText and a Payment ref
+     *
+     * @return A ForageAPIResponse indicating the success or failure of the
+     * payment capture. On success, returns a confirmation of the transaction.
+     * On failure, provides a detailed error response.
+     *
+     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
+     * hasn't had its ForageConfig set via .setForageConfig().
+     */
+    override suspend fun capturePayment(
+        params: CapturePaymentParams
+    ): ForageApiResponse<String> {
+        return forageSdk.capturePayment(params)
+    }
```

Comment:
Nit:  If you're into code golf 🏌️⛳, I think you can do this in Kotlin
```suggestion
    override suspend fun capturePayment(
        params: CapturePaymentParams
    ): ForageApiResponse<String> = ForageSdk().capturePayment(params)
```

Reply:
related https://github.com/teamforage/forage-android-sdk/pull/149/files#r1445307766